muxconfig/ui/ui_scrConfig.c

```diff
@@ -172,7 +172,7 @@ void ui_scrConfig_screen_init(void)
     lv_obj_set_height(ui_pnlContent, 400);
     lv_obj_set_x(ui_pnlContent, 0);
     lv_obj_set_y(ui_pnlContent, 34);
-    lv_obj_set_flex_flow(ui_pnlContent, LV_FLEX_FLOW_ROW_WRAP);
+    lv_obj_set_flex_flow(ui_pnlContent, LV_FLEX_FLOW_COLUMN);
```

Comment:
This needs to be set as a column otherwise when size to content shrink the items they up all in one row next to each other.  Column keeps each item as it's own row